config/dev/{% if use_ingestion and ingestion_framework == 'airbyte' %}airbyte.yml{% endif %}.tmpl

```diff
@@ -0,0 +1,110 @@
+airbyte_connection_id: airbyte_connection_id
+airbyte_url: https://airbyte.dataops-dev.getindata.dev
+connections:
+  POSTGRES_BQ_CONNECTION:
+    destinationId: [[ destination_id ]]
+    name: POSTGRES_BQ_CONNECTION
+    namespaceDefinition: customformat
+    namespaceFormat: jaffle_shop
```

Comment:
Would be neat to use variable from copier questions [[ dataset ]] for example rather than plain text and it can be used in here https://github.com/getindata/data-pipelines-template-example/blob/main/config/base/bigquery.yml.tmpl#L6